spec/jekyll-feed_spec.rb

```diff
@@ -267,12 +282,4 @@
       expect(contents).to include('<?xml-stylesheet type="text/xml" href="http://example.org/feed.xslt.xml"?>')
     end
   end
-
-  context "with site.lang set" do
-    let(:overrides) { { "lang" => "en-US" } }
-
-    it "should set the language" do
-      expect(contents).to match %r{type="text/html" hreflang="en-US" />}
```

Comment:
Perhaps I should leave this test, as it appears to be checking [something else](https://github.com/jekyll/jekyll-feed/blob/v0.9.1/lib/jekyll-feed/feed.xml#L8)